quinn-proto/src/connection/streams/recv.rs

```diff
@@ -381,27 +362,6 @@ pub enum ReadError {
     /// The stream has not been opened or was already stopped, finished, or reset
     #[error("unknown stream")]
     UnknownStream,
-    /// Attempted an ordered read following an unordered read
-    ///
-    /// Performing an unordered read allows discontinuities to arise in the receive buffer of a
-    /// stream which cannot be recovered, making further ordered reads impossible.
-    #[error("ordered read after unordered read")]
-    IllegalOrderedRead,
```

Comment:
Let's remove this case in the previous commit, where the `ReadableError` case that takes its place is introduced.

Reply:
There no longer is a previous commit? I think you may have reviewed some out-of-date commits as well as the current ones...

Reply:
Github is currently displaying two commits for this PR. "The previous commit" refers to 7b3953ae58f6bedefa2908ec6d7a327c036c59bf.

Reply:
On review, I think this comment is confusing because it's being displayed on the "all commits" view; the intended reading is on the second commit's diff, where `ReadError::IllegalOrderedRead` was removed.